apis/v1beta1/types.go

```diff
@@ -258,6 +258,11 @@ type PCIDeviceSpec struct {
 	// virtual machine is cloned.
 	// +kubebuilder:validation:Required
 	VendorID *int32 `json:"vendorId,omitempty"`
+	// Label is the hardware label of a virtual machine's PCI, in string.
+	// Defaults to the eponymous property value in the template from which the
+	// virtual machine is cloned.
+	// +optional
+	Label string `json:"label,omitempty"`
```

Comment:
```suggestion
	CustomLabel string `json:"customLabel,omitempty"`
```

Let's name it CustomLabel so it better maps to the underlying datastructure and does not get mixed up with kubernetes labels.

This then also maps to the name used in vm operator mode.

Reply:
Fixed.